app/assets/javascripts/events.js

```diff
@@ -0,0 +1,38 @@
+function eventCalendar() {
+  return $('#calendar').fullCalendar({
+    defaultView: 'listYear',
+    buttonText: {
+      today: 'Today'
+    },
+    eventRender: function (event, element, view) {
+      var description = event.description ? event.description : "";
+      var location = event.location ? event.location : "";
+      element.find(".fc-event-dot").css('display','none');
+      element.find('.fc-list-item-title').append('<div></div><span style="font-size: 12px">' + description + '</span>');
+      element.find('.fc-list-item-title').append('<div></div><span style="font-size: 12px">' + location + '</span>');
+    },
+    events: {
+      url: '/manage/events.json',
+      success: function(response) {
+        // due to "end" being a keyword in ruby and what fullcalender uses I store it as finish and than convert
```

Comment:
Good catch; try to avoid using "I" in comment so going forward code is more objective, e.g.: "... and what fullcander uses we store as finish, and then convert"